chat-with-tor/README.md

```diff
@@ -0,0 +1,15 @@
+# libp2p-chat-app
+Chat app demo based on @whyrusleeping's gist.
+
+# Instructions
+
+Follow the [tutorial](libp2p-demo-tutorial.md).
+
+Install deps and build:
+
+```shell
+$ make deps
```

Comment:
Given that we're not using gx, I'd almost just remove the make file and tell users to run:

```bash
> go get -u ./...
> go build # or maybe just `go run`?
```